server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala

```diff
@@ -62,13 +62,22 @@ class DeltaSharingServiceExceptionHandler extends ExceptionHandlerFunction {
     cause match {
       // Handle exceptions caused by incorrect requests
       case _: DeltaSharingNoSuchElementException =>
-        HttpResponse.of(
-          HttpStatus.NOT_FOUND,
-          MediaType.JSON_UTF_8,
-          JsonUtils.toJson(
-            Map(
-              "errorCode" -> ErrorCode.RESOURCE_DOES_NOT_EXIST,
-              "message" -> cause.getMessage)))
+        if (req.method().equals(HttpMethod.HEAD)) {
+          HttpResponse.of(
+            ResponseHeaders.builder(HttpStatus.NOT_FOUND)
+              .set("error-code", ErrorCode.RESOURCE_DOES_NOT_EXIST)
+              .set("message", cause.getMessage)
```

Comment:
perhaps not change these codes? I think it's not worth to add these non standard headers for this small change.

Reply:
Thank you for you comment.
I think that your opinion is correct.
I removed this two headers.